selfdrive/car/card.py

```diff
@@ -18,23 +18,26 @@
 
 REPLAY = "REPLAY" in os.environ
 
+State = log.ControlsState.OpenpilotState
 EventName = car.CarEvent.EventName
 
 
-class CarD:
+class Car:
```

Comment:
I think `card` is small enough that wrapping in this class makes it less readable with all the `self` and misdirection

Reply:
Will do after this is merged